packages/babel-types/src/definitions/flow.js

```diff
@@ -464,3 +465,80 @@ defineType("Variance", {
 defineType("VoidTypeAnnotation", {
   aliases: ["Flow", "FlowType", "FlowBaseAnnotation"],
 });
+
+// Enums
+defineType("EnumDeclaration", {
+  alises: ["Declaration"],
+  visitor: ["id", "body"],
+  fields: {
+    id: validateType("Identifier"),
+    body: validateType([
+      "EnumBooleanBody",
+      "EnumNumberBody",
+      "EnumStringBody",
+      "EnumSymbolBody",
+    ]),
+  },
+});
+
+defineType("EnumBooleanBody", {
+  visitor: ["members"],
+  fields: {
+    explicit: validate(assertValueType("boolean")),
+    members: validateArrayOfType("EnumBooleanMember"),
+  },
+});
+
+defineType("EnumNumberBody", {
+  visitor: ["members"],
+  fields: {
+    explicit: validate(assertValueType("boolean")),
+    members: validateArrayOfType("EnumNumberMember"),
+  },
+});
+
+defineType("EnumStringBody", {
+  visitor: ["members"],
+  fields: {
+    explicit: validate(assertValueType("boolean")),
+    members: validateArrayOfType(["EnumStringMember", "EnumDefaultedMember"]),
+  },
+});
+
+defineType("EnumSymbolBody", {
+  visitor: ["members"],
+  fields: {
+    members: validateArrayOfType("EnumDefaultedMember"),
+  },
+});
+
+defineType("EnumBooleanMember", {
+  visitor: ["id"],
+  fields: {
+    id: validateType("Identifier"),
+    init: validate(assertValueType("boolean")),
```

Comment:
This is inconsistent with the other types, it should be `BooleanLiteral`

Reply:
Will update.